roles/set_openstack_containers/README.md

```diff
@@ -18,6 +18,7 @@ The role will generate two 2 files in ~/ci-framework-data/artifacts/ directory a
 * `cifmw_set_openstack_containers_prefix`: Update container containing. Defaults to `openstack`
 * `cifmw_set_openstack_containers_openstack_version_change`: (Boolean) Set environment variables for openstack services containers for specific OPENSTACK_RELEASE_VERSION defined in cifmw_set_openstack_containers_update_target_version. It should be used only for meta openstack operator in prepare for openstack minor update. Defaults to `false`.
 * `cifmw_set_openstack_containers_update_target_version`: Value of OPENSTACK_RELEASE_VERSION env in openstack operator that should be set. Defaults to `0.0.2`.
+* `cifmw_set_openstack_containers_openstack_final_env`: File name to store the operator env in a file. Default to `operator_env.txt`.
```

Comment:
operator_env_after_update.txt

Reply:
...  so I'm not sure what you meant for default. in the update playbook you use operator_env_after_update.txt

ah so perhaps this is right. operator_env.txt in 'normal' deploy but you then call it something else during the update so you don't overwrite operator_env.txt ?